src/System.Runtime/tests/System/DateTime.cs

```diff
@@ -313,6 +313,16 @@ public static void TestSubtract_DateTime(DateTime dateTime1, DateTime dateTime2,
     }
 
     [Fact]
+    public static void TestSubtract_DateTime_Invalid()
+    {
+        DateTime date1 = DateTime.MinValue.ToLocalTime();
+        Assert.Throws<ArgumentOutOfRangeException>(() => date1.Subtract(new TimeSpan(365, 0, 0, 0)));
+
+        DateTime date2 = DateTime.MaxValue.ToLocalTime();
```

Comment:
I believe it's there to cover logic in ToLocalTime (DateTime.cs, line 1121, 1124) for when the converted ticks is less than/greater than the minimum/maximum allowed ticks.